src/util/os_utils.js

```diff
@@ -511,6 +511,8 @@ function get_ntp() {
                 let regex_res = (/server (.*) iburst #NooBaa Configured NTP Server/).exec(res);
                 return regex_res ? regex_res[1] : "";
             });
+    } else if (process.env.container === "docker") { //Explicitly returning on docker
```

Comment:
Maybe pull this to the top of the if-chain so it will override the linux case by being first